packages/typespec-ts/src/modular/static/scripts/renameOutput.mjs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as fs from "fs";
+import * as path from "path";
+import * as url from "url";
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const destinationPath = path.join(__dirname, "../dist/");
+
+const inputFile = path.join(destinationPath, "index.js");
+const inputMapFile = path.join(destinationPath, "index.js.map");
+
+if (fs.existsSync(inputFile) && fs.existsSync(inputMapFile)) {
+  fs.renameSync(inputFile, path.join(destinationPath, "index.cjs"));
+  fs.renameSync(inputMapFile, path.join(destinationPath, "index.cjs.map"));
```

Comment:
why we need to rename the output ?

Reply:
This is required to make it a hybrid package and we can export both versions cjs and ESModules

Reply:
Otherwise, users would be forced to migrate their projects to ESModules

Reply:
get it